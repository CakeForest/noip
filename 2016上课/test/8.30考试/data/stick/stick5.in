7
49 48 47 46 45 44 43
7
3 4 5 5 5 5 13
7
2 7 7 7 7 10 20
0