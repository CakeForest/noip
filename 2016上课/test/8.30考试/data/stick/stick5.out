322
20
30